lua/dapui/render/canvas.lua

```diff
@@ -170,6 +171,21 @@ function M.render_buffer(state, buffer)
       { end_col = pos[3] and (pos[2] + pos[3] - 1), hl_group = match[1] }
     )
   end
+  if expand_lines.enabled then
+    vim.cmd("augroup DAPUIExpandLongLinesFor" .. vim.fn.bufname(buffer):gsub('DAP ', ''))
+    vim.cmd("autocmd!")
+    vim.cmd(
+      "autocmd CursorHold <buffer="
```

Comment:
Since we're allowing a custom delay I think the `CursorMoved` event should be used at it'll allow for snappier results (my preference would be to instantly show it for example)

Reply:
It''s worth experiment to not split this delay to the `updatetime` and custom part.

Reply:
Zero delay will be my choice too :+1: 

Nvim 0.7 is released so we can remember to change those autocommands to theirs lua equivalents.